### PR TITLE
docs(multi-action-button): refactor multi-action-button stories - FE-3444

### PR DESCRIPTION
**Pull request by mkrds**
### Proposed behaviour

This PR refactors MultiActionButton to use `csf` / `mdx` formats instead of deprecated `storiesOf`

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

**Review by DipperTheDan — APPROVED** [review]

File: cypress/features/regression/multiActionButton.feature [hunk 1]
```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
...
-
-  @positive
-  Scenario: Disabled Multi Action Button component
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "disabled" object name
-    Then Multi Action Button state is disabled
-
-  @positive
-  Scenario: Enabled Multi Action Button component
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "disabledFalse" object name
-    Then Multi Action Button state is not disabled
```
Comment (KatarzynaQA) [inline cypress/features/regression/multiActionButton.feature:51]: remove this step from `multi-action-button-steps.js` file

File: cypress/features/regression/multiActionButton.feature [hunk 2]
```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
...
-    Then Multi Action Button align on preview is "<align>"
-    Examples:
-      | align | nameOfObject |
-      | left  | alignLeft    |
-      | right | alignRight   |
-
-  @positive
-  Scenario: Disabled Multi Action Button component
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "disabled" object name
-    Then Multi Action Button state is disabled
```
Comment (KatarzynaQA) [inline cypress/features/regression/multiActionButton.feature:46]: remove this step from `multi-action-button-steps.js` file

File: cypress/features/regression/multiActionButton.feature [hunk 3]
```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
...
-      | size   | height | nameOfObject |
-      | small  | 32     | sizeSmall    |
-      | medium | 40     | sizeMedium   |
-      | large  | 48     | sizeLarge    |
-
-  @positive
-  Scenario Outline: Check <align> property of Multi Action Button component
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
-      And I hover on Multi Action Button
-    Then Multi Action Button align on preview is "<align>"
```
Comment (KatarzynaQA) [inline cypress/features/regression/multiActionButton.feature:37]: remove this step from `multi-action-button-steps.js` file

File: cypress/features/regression/multiActionButton.feature [hunk 4]
```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
...
-      And Multi Action Button border color is "<border-color>" border-color
-    Examples:
-      | buttonType | background-color | border-color     | nameOfObject        |
-      | primary    | rgb(0, 129, 93)  | rgba(0, 0, 0, 0) | buttonTypePrimary   |
-      | secondary  | rgba(0, 0, 0, 0) | rgb(0, 129, 93)  | buttonTypeSecondary |
-
-  @positive
-  Scenario Outline: Set Multi Action Button size to <size>
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
-    Then Multi Action Button height is <height>
```
Comment (KatarzynaQA) [inline cypress/features/regression/multiActionButton.feature:26]: remove this step from `multi-action-button-steps.js` file

File: cypress/features/regression/multiActionButton.feature [hunk 5]
```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
...
     Examples:
       | text                         | nameOfObject         |
       | mp150ú¿¡üßä                  | textOtherLanguage    |
       | !@#$%^*()_+-=~[];:.,?{}&"'<> | textSpecialCharacter |
 
-  @positive
-  Scenario Outline: Change buttonType property of Multi Action Button component to <buttonType>
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
-    Then Multi Action Button has "<background-color>" background-color
-      And Multi Action Button border color is "<border-color>" border-color
```
Comment (KatarzynaQA) [inline cypress/features/regression/multiActionButton.feature:17]: remove this step from `multi-action-button-steps.js` file

Comment (DlgSHi) [reply cypress/features/regression/multiActionButton.feature:17]: can't remove those steps - are still used in other TC

**Review by KatarzynaQA — APPROVED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

**Review by nicktitchmarsh — DISMISSED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by Alec-Walker-Sage — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

**Pull request merged**